lib/net/nforceif/include/lwipopts.h

```diff
@@ -310,6 +310,24 @@
  */
 #define LWIP_TCP                        1
 
+/**
+ * TCP_MSS: TCP Maximum segment size. (chosen to be the maximum
+ * that can be transmitted over IPv6+PPPoE unfragmented)
+ * For the receive side, this MSS is advertised to the remote side
+ * when opening a connection. For the transmit size, this MSS sets
+ * an upper limit on the MSS advertised by the remote host.
+ */
+#define TCP_MSS                         1432
+
+/**
+ * TCP_WND: The size of a TCP window.  This must be at least
+ * (2 * TCP_MSS) for things to work well.
+ * ATTENTION: when using TCP_RCV_SCALE, TCP_WND is the total size
+ * with scaling applied. Maximum window value in the TCP header
+ * will be TCP_WND >> TCP_RCV_SCALE
+ */
+#define TCP_WND                         (45 * TCP_MSS)
```

Comment:
How did you arrive at these values? `TCP_MSS` and `TCP_WND`?

Reply:
`TCP_MSS` is set to the Ethernet frame size minus PPPoE, IPv6 and TCP headers. I wanted to support this configuration without risking fragmenting packets, while still having the MSS large enough to get good performance.

`TCP_WND` is set to the largest multiple of `TCP_MSS` (required by lwip) that fits into the 16 bit field (2^16 / 1432 = 45.765).

Reply:
I feel like such comments should be part of the config file / code.
It's not obvious but it does make sense.

Reply:
I added comments to those two entries now.